# Goal
geometry_msgs/Point goal_point # x, y, z
---
# Result
float32 elapsed_time # in seconds
---
# Feedback
float32 distance_to_point # in meters